### ISSUE TITLE
Lost tags

### ISSUE THREAD

**Issue by crusader1477** [DestinyItemManager/DIM#6746]
Using beta.
Switched between d2 and d1
When switching back lost all tags and all items were new items

**Comment by crusader1477** [comment]
I did have it on. Has been working fine for ages between my mac iPad and phone

**Comment by bhollis** [comment]
Hmm, well tags and new items are stored separately between D1 and D2 so I dunno.

**Comment by crusader1477** [comment]
Yeah no worries. I must have done something to wipe the synced data but can’t put my finger on anything different happening other than looking at d1 for the first time in forever. Bloody Hung Jury nostalgia :-)

Won’t take me long to retag and probably a good time to tidy up my vault
